### ISSUE TITLE
Bug: No data cells assigned to table header

### ISSUE THREAD

**Issue by pooriamehregan** [Informasjonsforvaltning/fdk-issue-tracker#669]
[No data cells assigned to table header - Issues - Accessibility NextGen - Accessibility.pdf](https://github.com/Informasjonsforvaltning/fdk-issue-tracker/files/10402103/No.data.cells.assigned.to.table.header.-.Issues.-.Accessibility.NextGen.-.Accessibility.pdf)

### Description
Every header cell in a table should have at least one data cell assigned to it.

### Learn more:
Data tables can present a challenge to people who are blind or have low vision — especially if the table contains a lot of information.While sighted visitors may be able to use visual cues to scan tables for information, people using screen readers can only do this if cells and headers are marked up correctly in the code

Gjelder følgende sider: 
https://data.norge.no/organizations/991825827/datasets/36f1732e-abae-3512-9978-bc382c7bc2cb
https://data.norge.no/organizations/974760673/datasets/68d08f28-a16d-4fab-a953-ed4ab08ce2e2
https://data.norge.no/organizations/971526920/datasets
https://data.norge.no/organizations/991825827/datasets
https://data.norge.no/organizations/999601391/datasets
https://data.norge.no/organizations/999601391/datasets/750a9ccd-30f0-3ff4-9fba-f91db37f6ba5
https://data.norge.no/organizations/970018131/datasets
https://data.norge.no/organizations/970205039/datasets
https://data.norge.no/organizations/974760673/datasets/18bb61d1-42a2-4c3c-8259-7470582e0b69
https://data.norge.no/organizations/999601391/datasets/1d29a7ab-042b-3b0f-a13b-23935977da9a
https://data.norge.no/organizations/974761076/datasets/e281c8c6-b944-4662-861d-a475e973e393
https://data.norge.no/organizations/974761076/datasets
https://data.norge.no/organizations/971526920/datasets/44f30e8d-b653-4463-9e78-73aa7fbcfdf0
https://data.norge.no/organizations/974760673/datasets/7c87f169-2520-4e56-ba2a-b7a3cc7de2e9
https://data.norge.no/organizations/974760673/datasets/8bbb82ba-9d2a-322b-a6cb-44f90fdc6916
https://data.norge.no/organizations/970205039/datasets/d3168c78-5b1f-445fb08b-c30a83f7b6f4
https://data.norge.no/organizations/970205039/datasets/7dae91e0-9e69-3319-9dcb-1e2765d81c32
https://data.norge.no/organizations/974760673/datasets

Alle disse sidene er knyttet til oversikter over metadatakvalitet. Er det noe vi ikke har gjort riktig i henhold til wcag i tabellene på disse sidene generelt? Si fra om dere trenger mer info

**Comment by KjerstiSteien** [comment]
Kan dette løses med å begynne en overgang til designsystemets table? Er portalsidene kompatible med den? Hvis ikke, hva må til for å gjøre portal kompatibel? https://storybook.designsystemet.no/?path=/docs/felles-table--docs

**Comment by pooriamehregan** [comment]
![image](https://github.com/Informasjonsforvaltning/fdk-issue-tracker/assets/50879748/b120cb3d-4fa3-4a09-a6c5-d3532b7942f0)

**Comment by pooriamehregan** [comment]
Testest med AXE, WAWE, w3 validator og Lighthouse. Har også testet manuelt ved å sjekke HTML-en visuelt, men fant ikke feil. 

Med tastaturnavigering og skjermleser så hopper de over tabellen (og oversikten på side) i noen tilfeller. Men tviler på at dette er årsaken fordi Siteimprove klarte å finne seg frem til tabellen (les feilbeskrivelsen).

Vi har ingen måter å teste eller være sikker på at disse feilene fortsatt gjelder. 

Flytter dette issuet til backlog inntil videre. 

Monsido kan hjelpe når den er klar for tilgjengelighetstesting.
